packages/core/src/components/Dialog/__stories__/Dialog.stories.tsx

```diff
@@ -338,7 +442,7 @@ export const HideTriggers = {
       <Flex
         data-testid={HIDE_TRIGGERS_CONTAINER}
         id={HIDE_TRIGGERS_CONTAINER}
-        className="monday-storybook-dialog_hide-triggers-story"
+        style={{ paddingInline: "var(--sb-spacing-small)" }}
```

Comment:
I actually rather it to be in px and not in vars because whoever copies it won't have that token